pyiron/atomistics/structure/atoms.py

```diff
@@ -239,6 +239,8 @@ def cell(self):
         numpy.ndarray: A size 3x3 array which gives the lattice vectors of the cell as [a1, a2, a3]
 
         """
+        if self._cell is None:
```

Comment:
Is not this redundant? `if self._cell is None`, then return `self._cell` returns `None`

Reply:
yes this because redundant again :D